src/ngraph/op/min.hpp

```diff
@@ -73,6 +73,7 @@ namespace ngraph
                           const Output<Node>& reduction_axes,
                           bool keep_dims = false);
 
+                using util::ArithmeticReductionKeepDims::visit_attributes;
```

Comment:
Why do you use using? It also will work without this.

Reply:
Yes, you are correct.
I removed using from all reduce ops.